util/fs/fs_test.go

```diff
@@ -99,6 +99,56 @@ func TestFSDetector(t *testing.T) {
 	}
 }
 
+
+func TestResize(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("TestFSDetector skipped because OS is not 'linux'")
+	}
+	if !hasPasswordlessSudo() {
+		t.Skip("TestFSDetector skipped because password-less sudo is required and not presently available")
+	}
+
+	// Non-existent test case.
+	if err := Resize("/TestFSDetector/foo/bar/baz"); err == nil  {
```

Comment:
Also, contrary to what I'd expected it seems like both the Formatting and Resize tests should be run together, since they're doing almost the same exact thing.